shell/platform/linux/BUILD.gn

```diff
@@ -220,6 +220,11 @@ executable("flutter_linux_unittests") {
 shared_library("flutter_linux_gtk") {
   deps = [ ":flutter_linux" ]
 
+  if (is_linux) {
+    # Ensure we can dlopen bundled dlls in bundle/lib on Flutter Desktop.
```

Comment:
The engine shouldn't reference bundling details that are controlled by the default runner template. Someone could use this library and not even use the template.